src/middlewared/middlewared/etc_files/local/nginx/nginx.conf.mako

```diff
@@ -56,6 +57,10 @@
             })
     ip6_list = [f'[{ip}]' for ip in ip6_list]
 
+    wg_config = middleware.call_sync('datastore.config', 'system.truecommand')
+    if wg_config['enabled'] and wg_config['wg_address']:
+        ip4_list.append(ipaddress.ip_network(wg_config['wg_address'], False).network_address)
```

Comment:
Dont you have to reload nginx after wireguard starts?

What happens if you start nginx and the ip is not there yet in the wireguard interface?

Reply:
Also for wireguard, we probably always want to listen on 80/443